src/config/main.js

```diff
@@ -48,6 +48,8 @@ const USE_EEG =
 const USE_PHOTODIODE =
   process.env.REACT_APP_USE_PHOTODIODE === "true" && USE_ELECTRON;
 
+const ACCURACY_CUTOFF = process.env.REACT_APP_MSIT_ACCURACY_CUTOFF || 50
```

Comment:
is the env var read in as a number? or does it need to be coerced?

Reply:
Yes, it's read as a number, so `REACT_APP_MSIT_ACCURACY_CUTOFF=30`, I tested those. It should be the same in production.